tests/exit-init/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#![cfg_attr(not(feature = "std"), feature(alloc_error_handler))]
+#![cfg_attr(not(feature = "std"), feature(const_btree_new))]
```

Comment:
Where do you use `BTreeMap::new()` here?

Reply:
Good catch: it's not used =)